pysat/tests/test_utils_files.py

```diff
@@ -296,3 +296,107 @@ def test_updating_directories(self, capsys):
 
         # Store new format like a typical user would
         pysat.params['directory_format'] = templ
+
+
+class TestFileUtils(CICleanSetup):
+    """Unit tests for general file/path utilities."""
+
+    def setup(self):
+        """Set up the unit test environment for each method."""
+        self.out = ''
+
+        # Use a two-year as default.  Some tests will use custom ranges.
+        self.start = dt.datetime(2008, 1, 1)
+        self.stop = dt.datetime(2009, 12, 31)
```

Comment:
Recommend we use the `pysat.instruments.pysat_testing` test_time dict to define these.